src/main/java/igrad/storage/JsonSerializableCourseBook.java

```diff
@@ -37,7 +42,13 @@ public JsonSerializableCourseBook(@JsonProperty("modules") List<JsonAdaptedModul
      * @param source future changes to this will not affect the created {@code JsonSerializableCourseBook}.
      */
     public JsonSerializableCourseBook(ReadOnlyCourseBook source) {
-        modules.addAll(source.getModuleList().stream().map(JsonAdaptedModule::new).collect(Collectors.toList()));
+        modules.addAll(source.getModuleList().stream()
+                           .map(JsonAdaptedModule::new)
+                           .collect(Collectors.toList()));
+        requirements.addAll(source.getRequirementList().stream()
```

Comment:
good you managed to figure out where the storage was, I didn't managed to figure that out, would reference this code then 😂